bundle/config/mutator/resourcemutator/run_as.go

```diff
@@ -129,6 +129,16 @@ func validateRunAs(b *bundle.Bundle) diag.Diagnostics {
 		))
 	}
 
+	// Secret Scopes do not support run_as in the API.
```

Comment:
The name doesn't need to be capitalized: https://docs.databricks.com/aws/en/security/secrets